tensorboard/notebook.py

```diff
@@ -145,7 +158,7 @@ def start(args_string):
     except ImportError:
         IPython = None
 
-    if context == _CONTEXT_NONE:
+    if context == _CONTEXT_NONE or context == _CONTEXT_CUSTOM:
```

Comment:
Why should we print `Launching TensorBoard...` when custom display function has been registered?

Reply:
This tells the user that the Tensorboard process is launching. It is standard behavior across the different python environments.